Do not overwrite FLAGS in order for the hardening to work.
--- a/src/GNUmakefile
+++ b/src/GNUmakefile
@@ -8,9 +8,9 @@
 INSTALL  = install
 CC       = gcc
 LD       = $(CC)
-CPPFLAGS = -I. -I$(VPATH)
-CFLAGS   = -Wall -Wextra -O2
-LDFLAGS  = -ltalloc
+CPPFLAGS += -I. -I$(VPATH)
+CFLAGS   += -Wall -Wextra -O2
+LDFLAGS  += -ltalloc
 
 OBJECTS = cli.o			\
 	notice.o		\
